downstream/modules/eda/ref-deploy-eda-controller-with-aap-operator-on-ocp.adoc

```diff
@@ -0,0 +1,7 @@
+[id="deploying-eda-controller-with-aap-operator-on-ocp"]
+
+= Deploying {EDA controller} with {OperatorPlatform} on {OCPShort}
```

Comment:
Remove space in attribute so it renders correctly. {EDAcontroller}

Reply:
done